.github/workflows/nightly.yml

```diff
@@ -37,7 +37,7 @@ jobs:
           echo "::set-output name=tag_name::${TAG_NAME}"
           echo "::set-output name=deploy_tag_name::deploy-${TAG_NAME}"
       - name: Deploy
-        uses: peaceiris/actions-gh-pages@v3
+        uses: peaceiris/actions-gh-pages@4eb285e828117bca26638192c3ed309c622e7bad
```

Comment:
apparently this commit is not a release and doesn't work at all. Maybe try the latest tagged release

Reply:
FWIW, I use this Bash function to find the SHA(s) of an Action's tag:

```
get_sha_for_action() {
    # Split the input into repo and tag
    repo="${1%@*}"        # Extracts everything before the '@'
    tag="${1#*@}"         # Extracts everything after the '@'

    # Fetch the tag information using curl and the GitHub API
    tag_info=$(curl -s -H "Accept: application/vnd.github.v3+json" \
    "https://api.github.com/repos/$repo/git/refs/tags/$tag")

    # Check if the tag is an annotated tag
    if echo "$tag_info" | grep -q '"type": "tag"'; then
        # Annotated tag
        tag_sha=$(echo "$tag_info" | jq -r '.object.sha')
        commit_sha=$(curl -s -H "Accept: application/vnd.github.v3+json" \
        "https://api.github.com/repos/$repo/git/tags/$tag_sha" | jq -r '.object.sha')

        echo "Annotated tag '$tag' found."
        echo "Tag SHA: $tag_sha"
        echo "Commit SHA: $commit_sha"
    else
        # Lightweight tag
        commit_sha=$(echo "$tag_info" | jq -r '.object.sha')

        echo "Lightweight tag '$tag' found."
        echo "Commit SHA: $commit_sha"
    fi
}
```